### DISCUSSION TITLE
Suggestion: matrix-item > titlebar > preview for lightswitch

### DISCUSSION THREAD

**Discussion by droid51** [craftcms/cms#8450] [Ideas]
Hi there,

small suggestion:

opened item
![grafik](https://user-images.githubusercontent.com/8912561/28706289-f23ddd22-7373-11e7-9d8d-9ad56aa8d516.png)

closed current
![grafik](https://user-images.githubusercontent.com/8912561/28706310-0804ddfe-7374-11e7-8eaa-e8b2d50df89c.png)

**closed should be**
![grafik](https://user-images.githubusercontent.com/8912561/28706426-9653ba30-7374-11e7-8f8b-417fb4a72a1b.png)

http://www.unicodemap.org/details/0x2610/index.html
http://www.unicodemap.org/details/0x2611/index.html

thx!
greetings crazyx13th

**Comment by Wiejeben** [comment]
How would you know what the lightswitch stands for, and what if there are multiple lightswitches?

**Comment by droid51** [comment]
mhh, maybe with a simple title attr with the "Entry-Name"
https://www.w3schools.com/TAGs/tryit.asp?filename=tryhtml5_global_title

or show "Entry-Name: ☑"

Thx!
